src/FsAutoComplete.Core/FsAutoComplete.Core.fsproj

```diff
@@ -223,4 +200,5 @@
     </When>
   </Choose>
   <Import Project="..\..\packages\FSharp.Compiler.Service.ProjectCracker\build\$(__paket__FSharp_Compiler_Service_ProjectCracker_targets).targets" Condition="Exists('..\..\packages\FSharp.Compiler.Service.ProjectCracker\build\$(__paket__FSharp_Compiler_Service_ProjectCracker_targets).targets')" Label="Paket" />
+  <Import Project="..\..\packages\NETStandard.Library\build\NETStandard.Library.targets" Condition="Exists('..\..\packages\NETStandard.Library\build\NETStandard.Library.targets')" Label="Paket" />
```

Comment:
`NETStandard.Library` shouldnt be needed there, everything dependency is `net40`, so there is a wrong restriction somewhere